src/Squidex/app/features/schemas/pages/schema/types/references-ui.component.html

```diff
@@ -18,5 +18,20 @@
                 <span class="radio-label">Dropdown</span>
             </label>
         </div>
-    </div>  
+    </div>
+
+    <div class="form-group row" *ngIf="field.properties.isContentField">
+        <div class="col-9 offset-3">
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" id="{{field.fieldId}}_fieldResolveReferences" formControlName="resolveReference" />
+                <label class="form-check-label" for="{{field.fieldId}}_fieldResolveReferences">
+                    Resolve reference
+                </label>
+            </div>
+
+            <sqx-form-hint>
+                If check the reference is resolved for the content list.<br />Can only be checken when the MaxItems field in the validation tab is set to 1.
```

Comment:
Can this sqx-form-hint be reworded slightly? 

Could be something like:
If 'Resolve Reference' is checked. <br /> This can only be checked when he 'Max Items' field in the validation tab is set to 1.